2019/July/README.md

```diff
@@ -1 +1,34 @@
-# talk links and description
+# July Python Pune meetup 2019
+  * [Event page on meetup.com](https://www.meetup.com/PythonPune/events/262769950/)
+
+## Python for simulating Poker Games
+  * **Speaker**: Abhijit Gadgil
+  * **Description**: This talk is about the proposal that is submitted
+    to Pycon India 2019. We look at simulating Poker games as a way of
+    getting started thinking in Python, that is in terms of it's
+    built-ins core constructs and standard libraries.
+
+  * [Slides](https://gabhijit.github.io/slides/pycon2019-poker/)
+  * [Source](https://github.com/gabhijit/pycon/tree/master/2019/poker)
+  * [Proposal](https://github.com/pythonpune/meetup-talks/issues/22)
+
+## Advanced Twitter Scraping using Twint
+  * **Speaker**: Sourav Singh
+  * **Description**: When we want to scrape tweets from Twitter, we
+    are limited by API limits. Twint is an advanced tool for scraping
+    tweets from Twitter without any need of an API key. Twint can also
+    do advanced scraping like scraping tweets based on a location,
+    scraping only tweets of a verified person on a topic and many
+    others. I hope to explore the tool in the short session.
```

Comment:
@bhavin192 about description last line... is it required as part of meetup talk doc?

Reply:
Yes, let's remove that.